lib/cadet/assessments/assessments.ex

```diff
@@ -12,44 +12,31 @@ defmodule Cadet.Assessments do
   alias Cadet.Assessments.Assessment
   alias Cadet.Assessments.Question
 
-  def all_assessments() do
-    Repo.all(Assessment)
-  end
-
   def all_assessments(assessment_type) do
```

Comment:
I think we should wait until my PR #101 is merged cos I added max_xp virtual field as well as query to fetch that.

Reply:
Do you wanna add the `max_xp` field here?

Reply:
tbh i'm thinking of removing these methods. nothing's using them right now

Reply:
Haha yes. I don't think anyone would miss it

Reply:
Actually I'm thinking if it's possible to refactor `Assessments` context such that we can remove the `assessments_test.exs` altogether and getting coverage just from the controller tests. What do you think?

Reply:
We can do away with a lot of them but we still need the functions that build questions/assessments